src/test/java/com/google/api/codegen/ProtoDocumentLinkTest.java

```diff
@@ -47,22 +47,23 @@ public void testMatchProtoLink() {
   }
 
   @Test
-  public void testRDocCommentFixer() {
```

Comment:
Sure the logic has not been changed, but leaving it as is until MVVM deletes the RDocComment fixer protects us on the (admittedly very) off chance, between the time this PR is merged and the time MVVM is, from a bug being introduced in the RDocCommentFixer.

I agree, the same logic is continuing to be tested, so I'm fine with changing the tests now but would prefer changing them during the MVVM PR. Your call.